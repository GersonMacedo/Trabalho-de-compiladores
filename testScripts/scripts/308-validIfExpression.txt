! -s CONTENT --disable-logs printer
! should accept a boolean expression in the if
program p ;
    var b1 : boolean;
begin
    if b1 then
        b1 := true
    ;
end .