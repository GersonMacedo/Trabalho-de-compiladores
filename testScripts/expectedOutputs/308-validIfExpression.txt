
Performing up to content analysis at testScripts/scripts/308-validIfExpression.txt
No errors found
